service.audio.profiles/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.audio.profiles"
+    name="Audio Profiles Service"
+    version="1.0.0"
+    provider-name="Regss">
+    <requires>
+        <import addon="xbmc.python" version="2.1.0" />
+    </requires>
+    <extension point="xbmc.service" library="default.py" start="[login|startup]" />
```

Comment:
start="[login|startup]" is not valid ;-)

use either start="login" or start="startup".
see: http://kodi.wiki/view/Service_addons